### ISSUE TITLE
Dashboard: Links with same text

### ISSUE THREAD

**Issue by qanitah** [mmiron-HC/WST-RSAMS#55]
URL: https://hcdev.openplus.ca/en/user/203 (Dashboard)
User Agent: Chrome, Firefox, iPhone 12 (iOS 14.7.1)
Language: English

Issue Description: When there are more than one bookmarks added to this page, all links have text "View". Someone using screen reader and just going by links on the page will not know where the links are going to as there isn't any other info.

Same also applies to other content that repeats as well. For example, Submit, Apply and Build buttons, They all have multiple instances on the page. They should have a bit more information in the links for screen reader users.

Steps to reproduce: Add a few bookmarks from the Search tab. Go through the page using screen reader - navigate just links on the page.

Expected results: There should be more added to this link so the user would know where the link is going. The purpose of each link can be determined from the link text alone or from the link text together with its programmatically determined link context, 
There are several ways of doing this. https://dequeuniversity.com/rules/axe/3.5/link-name

![image](https://user-images.githubusercontent.com/14280062/128915057-68fb9ed5-8782-4b07-8dbf-1cc4751035ad.png)

**Comment by qanitah** [comment]
It now reads the page it will be going to with the screen reader.
